src/main/java/com/comphenix/protocol/utility/SchedulerUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.comphenix.protocol.utility;
+
+import com.comphenix.protocol.reflect.accessors.Accessors;
+import com.comphenix.protocol.reflect.accessors.MethodAccessor;
+import org.bukkit.Bukkit;
+import org.bukkit.plugin.Plugin;
+
+import java.util.function.Consumer;
+
+public class SchedulerUtil {
+    private Object foliaScheduler;
+    private MethodAccessor runAtFixedRate;
+    private MethodAccessor cancelTasks;
+    private MethodAccessor execute;
+
+    private static SchedulerUtil getInstance() {
+        return Holder.INSTANCE;
+    }
+
+    private static class Holder {
+        private static final SchedulerUtil INSTANCE = new SchedulerUtil();
+    }
+
+    private SchedulerUtil() {
+        if (Util.isUsingFolia()) {
+            MethodAccessor getScheduler = Accessors.getMethodAccessor(Bukkit.getServer().getClass(), "getGlobalRegionScheduler");
+            foliaScheduler = getScheduler.invoke(Bukkit.getServer());
+
+            runAtFixedRate = Accessors.getMethodAccessor(foliaScheduler.getClass(), "runAtFixedRate", Plugin.class,
+                    Runnable.class, long.class, long.class);
```

Comment:
```suggestion
                    Consumer.class, long.class, long.class);
```

https://jd.papermc.io/folia/1.19/io/papermc/paper/threadedregions/scheduler/GlobalRegionScheduler.html#runAtFixedRate(org.bukkit.plugin.Plugin,java.util.function.Consumer,long,long)

Runnable does not exist in `runAtFixedRate`. It's instead a `Consumer<ScheduledTask>`